segmentation/utils/spatial_analysis.py

```diff
@@ -46,15 +49,16 @@ def calculate_channel_spatial_enrichment(dist_matrix, marker_thresholds, all_pat
             variables are:
             z, muhat, sigmahat, p, h, adj_p, and marker_titles for each point in the analysis"""
 
+    patient_col = "SampleID"
```

Comment:
In the matlab script, we have "patient" represent each unique image, because there's only one image per patient. However, moving forward, we're going to have multiple images per patient, but we'll still want to do the analysis separately for each image. 

So, I think we should move away from patient as the keyword indicate unique image. Let's rename things as follows. 

all_data: this is the input matrix with everything. 
fovs (field of view, aka image): this will replace points everywhere
fov_data: this is what we'll call the subsetted matrix that contains only the current fov
fov_channel_data: this is what we'll call the array with the marker information
fov_cluster_data: this is what we'll call the array with the cell type information.